# Nicolas Ternisien <nicolas@bureau-cornavin.com>, 2016.
# Éric Bischoff <ebischoff@bureau-cornavin.com>, 2016.
msgid ""
msgstr ""
"Project-Id-Version: univention-management-console-module-adconnector\n"
"Report-Msgid-Bugs-To: packages@univention.de\n"
"POT-Creation-Date: 2021-11-03 17:54+0100\n"
"PO-Revision-Date: 2021-12-07 15:44+0100\n"
"Last-Translator: Nicolas Sidi <nicolas.sidi@mailbox.org>\n"
"Language-Team: French <fr@bureau-cornavin.com>\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
"X-Generator: Poedit 3.0\n"

#: umc/python/adconnector/__init__.py:331
msgid "Active Directory Connector is already running. Nothing to do."
msgstr ""
"Le connecteur Active Directory est déjà lancé. Aucune action à effectuer."

#: umc/python/adconnector/__init__.py:333
msgid "Active Directory Connector is already stopped. Nothing to do."
msgstr ""
"Le connecteur Active Directory est déjà arrêté. Aucune action à effectuer."

#: umc/python/adconnector/__init__.py:349
msgid "Active Directory connection service has been started."
msgstr "Le service de connexion Active Directory a été démarré."

#: umc/python/adconnector/__init__.py:351
msgid "Active Directory connection service has been stopped."
msgstr "Le service de connexion Active Directory a été arrêté."

#: umc/python/adconnector/__init__.py:277
msgid ""
"Active Directory connection settings have been saved and a new certificate "
"for the Active Directory server has been created."
msgstr ""
"Les réglages de la connexion Active Directory ont été enregistrés et un "
"nouveau certificat pour le serveur Active Directory a été créé."

#: umc/python/adconnector/__init__.py:268
msgid "Active Directory connection settings have been saved."
msgstr "Les réglages de la connexion Active Directory ont été enregistrés."

#: umc/python/adconnector/__init__.py:499
msgid "An error occurred while installing necessary software components."
msgstr "Erreur lors de l'installation des composants logiciels nécessaires."

#: umc/python/adconnector/__init__.py:455
#, python-format
msgid "An unexpected error occurred: %s"
msgstr "Une erreur inattendue est survenue : %s"

#: umc/python/adconnector/__init__.py:293
msgid "Certificate has been uploaded successfully."
msgstr "Le certificat a été envoyé avec succès."

#: umc/python/adconnector/__init__.py:302
msgid "Certificate upload or conversion failed."
msgstr "Échec de l'envoi ou de la conversion du certificat."

#: umc/python/adconnector/__init__.py:508
msgid "Configuring Administrator account..."
msgstr "Configuration du compte Administrator…"

#: umc/python/adconnector/__init__.py:514
msgid "Configuring DNS entries..."
msgstr "Configuration des enregistrements DNS..."

#: umc/python/adconnector/__init__.py:473
msgid "Configuring DNS server..."
msgstr "Configuration du serveur DNS..."

#: umc/python/adconnector/__init__.py:477
msgid "Configuring Kerberos settings..."
msgstr "Configuration des réglages Kerberos..."

#: umc/python/adconnector/__init__.py:481
msgid "Configuring reverse DNS settings..."
msgstr "Configuration des réglages DNS inversés..."

#: umc/python/adconnector/__init__.py:484
msgid "Configuring software components..."
msgstr "Configuration des composants logiciels..."

#: umc/python/adconnector/__init__.py:501
msgid "Configuring synchronization from AD..."
msgstr "Configuration de la synchronisation depuis AD..."

#: umc/python/adconnector/__init__.py:469
msgid "Configuring time synchronization..."
msgstr "Configuration de la synchronisation du temps..."

#: umc/python/adconnector/__init__.py:405
#: umc/python/adconnector/__init__.py:534
#, python-format
msgid ""
"Could not connect to AD Server %s. Please verify that the specified address "
"is correct. (%s)"
msgstr ""
"Impossible de se connecter au serveur AD %s. Assurez-vous que l’adresse "
"spécifiée est correcte. (%s)"

#: umc/python/adconnector/__init__.py:411
#: umc/python/adconnector/__init__.py:538
#, python-format
msgid ""
"Could not connect to AD Server %s. Please verify that username and password "
"are correct. (Details:\n"
"%s)"
msgstr ""
"Impossible de se connecter au serveur AD %s. Assurez-vous que l’identifiant "
"et le mot de passe sont corrects. (détails :\n"
"%s)"

#: umc/python/adconnector/__init__.py:583
#, python-format
msgid ""
"Could not establish an encrypted connection. Either \"%r\" is not reachable "
"or does not support encryption."
msgstr ""
"Impossible d'établir une connexion chiffrée.  Soit on ne peut pas accéder à "
"« %r », soit il ne gère pas le chiffrement."

#: umc/python/adconnector/__init__.py:298
msgid ""
"Could not establish connection. Either the certificate is wrong, the Active "
"Directory server is unreachable or it does not support SSL."
msgstr ""
"Impossible d'établir la connexion. Soit le certificat est incorrect, soit le "
"domaine Active DIrectory est inaccessible ou ne gère pas SSL."

#: umc/python/adconnector/__init__.py:550
#, python-format
msgid ""
"Could not synchronize the time between the UCS system and the Active "
"Directory domain controller: %s"
msgstr ""
"Impossible de synchroniser le temps entre le système UCS et le contrôleur du "
"domaine Active Directory : %s"

#: umc/python/adconnector/__init__.py:276
#, python-format
msgid "Creation of certificate failed (%s)"
msgstr "Échec de la création du certificat (%s)"

#: umc/python/adconnector/__init__.py:548
msgid "Failed to add the Administrator account to the Domain Admins group."
msgstr "Échec de l’ajout du compte Administrator au groupe Domain Admins."

#: umc/python/adconnector/__init__.py:542
msgid "Failed to create the Administrator account in UCS."
msgstr "Échec de la création du compte Administrator dans UCS."

#: umc/python/adconnector/__init__.py:546
msgid "Failed to search for the well known SID."
msgstr "Échec de la recherche du SID connu."

#: umc/python/adconnector/__init__.py:540
msgid ""
"Failed to set the password of the UCS Administrator to the Active Directory "
"Administrator password."
msgstr ""
"Impossible de fixer le mot de passe du compte Administrator d’UCS sur celui "
"du compte d’administration d’Active Directory."

#: umc/python/adconnector/__init__.py:528
msgid "Join has been finished successfully."
msgstr "La jonction s'est terminée correctement."

#: umc/python/adconnector/__init__.py:559
msgid "Join has been finished with errors."
msgstr "La jonction s'est terminée avec des erreurs."

#: umc/python/adconnector/__init__.py:432
msgid "Joining UCS into Active Directory domain"
msgstr "UCS rejoint le domaine Active Directory"

#: umc/python/adconnector/__init__.py:524
msgid "Registering LDAP service entry..."
msgstr "Enregistrement d'une entrée de service LDAP..."

#: umc/python/adconnector/__init__.py:505
msgid "Renaming well known SID objects..."
msgstr "Renommage de tous les objets SID connus..."

#: umc/python/adconnector/__init__.py:511
msgid "Running Samba join script..."
msgstr "Lancement du script Samba pour rejoindre le domaine..."

#: umc/python/adconnector/__init__.py:244
#, python-format
msgid "Saving bind password failed (filename=%(fn)s ; exception=%(exception)s)"
msgstr ""
"Échec de l'enregistrement du mot de passe de bind (nom du fichier=%(fn)s ; "
"exception=%(exception)s)"

#: umc/python/adconnector/__init__.py:521
msgid "Starting Active Directory connection service..."
msgstr "Démarrage du service de connexion à Active Directory..."

#: umc/python/adconnector/__init__.py:345
msgid "Switching running state of Active Directory Connector failed."
msgstr "Échec du passage du connecteur Active Directory en mode d'exécution."

#: umc/python/adconnector/__init__.py:402
#: umc/python/adconnector/__init__.py:532
msgid "The AD member mode can only be configured on a Primary Directory Node."
msgstr ""
"Le mode membre AD ne peut être configuré que sur un Primary Directory Node."

#: umc/python/adconnector/__init__.py:408
#: umc/python/adconnector/__init__.py:536
#, python-format
msgid ""
"The domain name of the AD Server (%(ad_domain)s) does not match the local "
"UCS domain name (%(ucs_domain)s). For the AD member mode, it is necessary to "
"setup a UCS system with the same domain name as the AD Server."
msgstr ""
"Le nom de domaine du serveur AD (%(ad_domain)s) ne correspond pas au nom de "
"domaine local d'UCS (%(ucs_domain)s). Pour le mode membre AD, il faut "
"configurer un système UCS avec le même nom de domaine que celui du serveur "
"AD."

#: umc/python/adconnector/__init__.py:414
msgid ""
"The given user is not member of the Domain Admins group in Active Directory. "
"This is a requirement for the Active Directory domain join."
msgstr ""
"L'utilisateur donné n'est pas membre du groupe Domain Admins dans Active "
"Directory. Ceci est un prérequis pour faire rejoindre le domaine Active "
"Directory."

#: umc/python/adconnector/__init__.py:544
msgid "The sambaSID could not set for the Administrator account in UCS."
msgstr ""
"Le sambaSID n’a pas pu être défini pour le compte Administrator dans UCS."
